Some performance tips (#792)
PrestaShop/docs: src/content/1.7/scale/_index.md

Also, once you have set up your first server, setting additional ones should be far easier.

The good thing is that, though requiring more sweat at first, horizontal scaling is better in terms of performances. Done well, adding a second server may easily double your application performance, if not more.

### Further than LAMP

Another way to scale your shop is to introduce other systems to improve your performances. Here is a quicklist of what can be used:

1. Using a reverse proxy such as varnish for Front Office
2. Using Redis to store sessions
3. Use elasticsearch or a solution built on it for processing customer searches

> 4. Use a CDN for static files

Reviewer: On a development environment an http cache can be tedious especially for front-end developers. I wonder if it is useful to specify that it is recommended in a production environment only?

Author: It's always good to be more accurate 👍

Author: Fixed in https://github.com/PrestaShop/docs/pull/792/commits/17de842062f4228fc27a9eebfc3513b43c0b04e4

Also, once you have set up your first server, setting additional ones should be far easier.

The good thing is that, though requiring more sweat at first, horizontal scaling is better in terms of performances. Done well, adding a second server may easily double your application performance, if not more.

### Further than LAMP

Another way to scale your shop is to introduce other systems to improve your performances. Here is a quicklist of what can be used:

> 1. Using a reverse proxy such as varnish for Front Office

Reviewer 2: ```suggestion
1. Using a HTTP accelerator such as varnish for Front Office
```
If people are looking for reverse proxy, it will not necessarily increase performance.

> 2. Using Redis to store sessions

Reviewer 2: ```suggestion
2. Using an in-memory data structure store such as Redis to store sessions
```

> 3. Use elasticsearch or a solution built on it for processing customer searches

Reviewer 2: ```suggestion
3. Use Elasticsearch or a solution built on it for processing customer searches
```

> 4. Use a CDN for static files

Reviewer 2: ```suggestion
4. Use a Content Delivery Network (CDN) for static files
```

> Note: services like a CDN or reverse proxy are useful in a production environment, not in a development environment.

Reviewer 2: ```suggestion
Note: services like a CDN or HTTP accelerator are useful in a production environment, not in a development environment.
```
